src/Microsoft.ML.Core/Data/IEstimator.cs

```diff
@@ -225,7 +225,7 @@ internal bool TryFindColumn(string name, out Column column)
     /// The 'data reader' takes a certain kind of input and turns it into an <see cref="IDataView"/>.
     /// </summary>
     /// <typeparam name="TSource">The type of input the reader takes.</typeparam>
```

Comment:
>reader [](start = 56, length = 6)

loader - I would do a pass over the comments replacing reader with loader.

Reply:
Thank you!


---
In reply to: [260394231](https://github.com/dotnet/machinelearning/pull/2731#discussion_r260394231) [](ancestors = 260394231)